docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
     tmpfs:
       - /myapp/tmp
     ports:
-      - "4200:4200"
+      - "80:4200"
```

Comment:
You really want to map your host port 80 to the containers server?

Reply:
As discussed, fixed by removing the `ember` container from the compose setup.